src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+
+  private String color;
+  private int number;
```

Comment:
Pay attention to access modifiers. Fields should be private to adhere to the principle of encapsulation. In this case, they are private which is good practice.